concourse/model/traits/image_scan.py

```diff
@@ -161,6 +161,24 @@ class ClamAVRescoringEntry:
         type=int,
         default=8,
     ),
+    AttributeSpec.optional(
+        name='virus_definition_age_threshold',
+        doc=(
+            'The maxmimum age difference (in days) the virus definition database of ClamAV may '
+            'have when being compared to the most current database before triggering rescans.'
+        ),
+        type=int,
+        default=0,
+    ),
+    AttributeSpec.optional(
+        name='skip_if_virus_definition_unchanged',
```

Comment:
I think there is not need to make this configurable.

if the data to scan did not change (e.g. determined through hash digest), and neither did the virus-db, it is always pointless to rescan (thus I think we should simply hard-code this case).

You however do have a valid point in having identified this case. Unchanged db-version should overrule exceeded max. allowed age